docs/tutorial-solver.md

```diff
@@ -138,7 +127,7 @@ root@d792c990d8bd:/app/dex-services# cargo run --bin driver
 2020-09-07T13:29:38.048Z INFO [services_core::history::events] Successfully loaded 60846 events in 12437006 bytes from event registry file
 2020-09-07T13:29:38.050Z INFO [services_core::orderbook::streamed::updating_orderbook] successfully recovered orderbook from path
 2020-09-07T13:29:38.284Z INFO [services_core::orderbook::streamed::updating_orderbook] Updating event based orderbook from block 7155117 to block 7155296.
-2020-09-07T13:29:40.031Z INFO [services_core::orderbook::streamed::updating_orderbook] Received 164 events.
+2020-09-07T13:29:40.031Z INFO [services_core::orderbook::streamed::updating_orderbootk] Received 164 events.
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F3ohzdYJK1wAdPWVk88%2Fgiphy.gif&token=oops" data-gitmeme-token="oops"><img src="https://media2.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif" title="Created by gitme.me with /oops"/></a>

Reply:
must have been one of those `git ...` with the wrong focus. Actually, you've reminded me that this snippet from the logs is outdated anyway!